internal/core/operations/switch_active_version/switch_active_version_executor_test.go

```diff
@@ -0,0 +1,382 @@
+// MIT License
+//
+// Copyright (c) 2021 TFG Co
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+package switch_active_version_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/topfreegames/maestro/internal/core/operations/switch_active_version"
+
+	"github.com/topfreegames/maestro/internal/core/services/interfaces"
+	mockeventsservice "github.com/topfreegames/maestro/internal/core/services/interfaces/mock/events_service"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/require"
+	clock_mock "github.com/topfreegames/maestro/internal/adapters/clock/mock"
```

Comment:
@caiordjesus Just a simple refactor, can we rename these import alias to remove underscores? we are now following the golang convention, ref:  https://youtrack.jetbrains.com/issue/GO-5942, https://github.com/kubernetes/kubernetes/pull/35258/files